docs/man/Dockerfile.5.md

```diff
@@ -117,6 +117,11 @@ or
  interconnect containers using links, and to set up port redirection on the host
  system.
 
+**UNEXPOSE**
+ --**UNEXPOSE <port> [<port>...]**
+ The **UNEXPOSE** instruction informs Docker that the container no
+ longer wants to listen on the specified network ports at runtime.
```

Comment:
Given that containers aren't sentient (yet), they can't "want" things. :-) Need to find a more accurate verb. Perhaps something like "...informs Docker that the container will no longer listen on the specified..."